crates/edr_rpc_server/src/node.rs

```diff
@@ -258,6 +246,28 @@ impl Node {
         node_data.remove_filter::</* IS_SUBSCRIPTION */ true>(filter_id).await
     }
 
+    pub async fn send_transaction(
+        &self,
+        transaction_request: EthTransactionRequest,
+    ) -> Result<B256, NodeError> {
+        let mut node_data = self.lock_data().await;
+
+        let signed_transaction = node_data.get_signed_transaction(transaction_request)?;
+
+        // TODO discuss in review: this can block the async executor if the state is remote or
+        // forked. Do we want to do something about in this PR or should we add a follow up issue?
+        node_data.add_pending_transaction(signed_transaction)
```

Comment:
Review question

Reply:
Feel free to create an issue for this. I think there are more places than just here, where we could employ the approach of [`spawn_blocking`](https://docs.rs/tokio/latest/tokio/task/fn.spawn_blocking.html) to prevent stalling.

Reply:
Thanks, in the meantime I looked into `tokio::task::block_in_place` once more and learned that it actually turns a worker thread into a blocking thread if it's called from a worker thread which is pretty cool and makes my concerns moot. Sorry for the false alarm!